docs/_data/atlantic-2023-config.yaml

```diff
@@ -88,7 +89,7 @@ sponsorship:
 date:
   main: "**September 10-12, 2023, online**"
   short: Sep 10-12, 2023
-  tickets_live: "June 2023"
+  tickets_live: "May 2023"
```

Comment:
We could bring this forward?

Reply:
Yes, I suppose we could move it to April.